src/makefiles/cuda_64bit.mk

```diff
@@ -10,7 +10,7 @@ CXXFLAGS += -DHAVE_CUDA -I$(CUDATKDIR)/include -fPIC -pthread -isystem $(OPENFST
 CUDA_INCLUDE= -I$(CUDATKDIR)/include -I$(CUBROOT) -I.. -isystem $(OPENFSTINC)
 CUDA_FLAGS = --compiler-options -fPIC --machine 64 -DHAVE_CUDA \
              -ccbin $(CXX) -DKALDI_DOUBLEPRECISION=$(DOUBLE_PRECISION) \
-             -std=c++14 -DCUDA_API_PER_THREAD_DEFAULT_STREAM -lineinfo \
+             -std=c++11 -DCUDA_API_PER_THREAD_DEFAULT_STREAM -lineinfo \
```

Comment:
is this change supposed to be here?

Reply:
With the `devtoolset-7-gcc` of the CentOS Developer Toolset, when I uses `c++14`, the building fails:
https://paste.ubuntu.com/p/N2xWXZ2jtm/

I haven't found why, but using `c++11` could address this problem. :flushed: